### DISCUSSION TITLE
Version 2.5 and 2.4 on different machines

### DISCUSSION THREAD

**Discussion by EddyWouldGo** [abraunegg/onedrive#2837] [General]
I have two different version running (2.4.25 on laptop, 2.5.0 on workstation). I use KDE Neon (Ubuntu 22.04) on both machines. I suspect that this is the result of performing a compiled-from-source install on one machine and installing from the OpenSuSE Build Service using the Ubuntu Package Install Instructions on the other. 

Anyway, I am taking seriously the admonition to not mix versions 2.4 and 2.5. 
From https://github.com/abraunegg/onedrive/blob/master/docs/install.md#upgrading-the-client, 
it seems clear that the OpenSUSE install must have gotten got the 2.5 and the compiled version stayed with the 2.4. So, the maybe all-too-obvious answer to the problem is to uninstall 2.4 on the compiled machine and reinstall using the OpenSUSE directions.  I just wanted to check my logic. Have i got this right or am I missing something major?

Please and thank you.

**Comment by abraunegg** [comment]
@EddyWouldGo 
> I have two different version running (2.4.25 on laptop, 2.5.0 on workstation).

Please read this *very* carefully: https://github.com/abraunegg/onedrive/releases

There are major differences - many around how the integrity of your data is handled - do not use both clients at the same time.

Either downgrade to 2.4.25 on all machines, or upgrade to 2.5.0 on all machines.

My advice for the moment, there are a number of bugs (despite >9 months community testing) that to me are show stoppers. I would downgrade | manually compile v2.4.25 and stay there for the moment until v2.5.1 comes out.

**Reply by EddyWouldGo** [reply]
Thanks much. I did read carefully, which is how I knew not to mix and match. I'll stick with 2.4.5 for now. You are awesome.
